core/src/main/java/com/spotify/ffwd/debug/DebugOutputPlugin.java

```diff
@@ -48,7 +50,7 @@ public Module module(final Key<PluginSink> key, final String id) {
             protected void configure() {
                 final Key<DebugPluginSink> sinkKey =
                     Key.get(DebugPluginSink.class, Names.named("debugSink"));
-                bind(sinkKey).to(DebugPluginSink.class);
+                bind(sinkKey).to(DebugPluginSink.class).in(Scopes.SINGLETON);
```

Comment:
this limits possibility to declare multiple debug output plugins

Reply:
This was a deliberate decision on the input of @udoprog. I believe that the reasoning is that there's no need to have multiple such output plugins specified.